_source/_includes/log-shipping/multiline-logs-filebeat.md

```diff
@@ -0,0 +1,81 @@
+## Multiline logs
+
+
+If your original logs span multiple lines, you may find that they arrive in your Logz.io account split into several partial logs. You have the option to use either an explicit configuration or an annotation to configure Filebeat to concatenate multiline logs and avoid having them separated into multiple log documents.
+
+You will need to provide a regex expression that can identify the first log in a multiline log. Filebeat will concatenate the log lines that follow until it identifies the next log that matches the regex expression. In other words, there is no explicit regex expression to demarcate the _end_ of a multiline log.
+
+This behavior is managed differently depending on your deployment method:
+
+* If you are using autodiscover hints & annotations, add an annotation to your deployment.
+* If you are not using autodiscover, use an explicit configuration.
+
+
+
+### Using an explicit configuration to concatenate multiline logs
+
+Open your `filebeat.yml` configuration file in a text editor. Under `filebeat.input`, add a multiline configuration to identify the first line in a multiline log. Filebeat will concatenate all subsequent lines until the `multiline.pattern` identifies the next match. For example:
+
```

Comment:
Same here like in the first comment, it's right for this specific example, but not generally, so maybe it's worth a rephrase or to make it clear that it behaves like this because we configured it to like this, but they can also configure it to behave in different ways, depending on their needs.